src/Providers/FractalProvider.php

```diff
@@ -8,7 +8,10 @@
 
 class FractalProvider extends AbstractServiceProvider
 {
-    protected $provides = [ Manager::class ];
+    protected $provides = [
+        'baseUrl',
```

Comment:
I think this is wrong..?

Reply:
Yep, remnant of a half-baked approach I abandoned. Will nuke!